### PR TITLE
Fix firmware installation for vfs

### LINKED ISSUE #3023: Install Firmware does not respect Virtual File System folder redir
1. Use Virtual Filesystem to point rpcs3 to a folder other than the default
2. Install the firmware
3. Try to launch a game, only for the emulator to not find the files due to them being placed in the dev_flash folder in the Emu Dir

### PR DESCRIPTION
**Pull request by Megamouse**
should fix #3023

### CODE REVIEW

**Comment by Nekotekina** [comment]
Oh noes, it breaks other cases

File: rpcs3/Emu/System.cpp [hunk 1]
```diff
@@ -509,11 +507,14 @@ void Emulator::Load(bool add_only)
 				return;
 			}
 
-			// Store /dev_bdvd/ location
-			games[m_title_id] = bdvd_dir;
-			YAML::Emitter out;
-			out << games;
-			fs::file(fs::get_config_dir() + "/games.yml", fs::rewrite).write(out.c_str(), out.size());
+			if (m_path.find(hdd0_disc) == -1)
```
Comment (Nekotekina) [inline rpcs3/Emu/System.cpp]: It'll break things. BDVD path should always be added there.

Comment (Megamouse) [reply rpcs3/Emu/System.cpp]: huh? i thought the games.yml was meant for external games only

**Comment by Megamouse** [comment]
removed the games.yml commit since it was not what we were looking for

**Pull request merged**